page.codeberg.zynequ.Pingveno.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id" : "page.codeberg.zynequ.Pingveno",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "pingveno",
+    "finish-args" : [
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+	      "--filesystem=xdg-download"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "pingveno",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://codeberg.org/zynequ/Pingveno/commit/dd777fb222c012e0e3f4fb830998f81ae9f4ced5"
```

Comment:
not sure what documentation you are following but this is not how you specify a commit.